velox/exec/FilterProject.cpp

```diff
@@ -150,23 +140,13 @@ RowVectorPtr FilterProject::getOutput() {
     evalCtx.ensureFieldLoaded(fieldIdx, *rows);
   }
 
+  std::vector<VectorPtr> results;
```

Comment:
nit: move this variable inside if + add a new only just before if (!isIdentityProjection_) {

This would make the code a bit more readable.